modules/tiles/test/tileset/tile-3d.spec.js

```diff
@@ -113,62 +117,72 @@ const MOCK_TILESET = {
 //   return Matrix4.pack(transformMatrix, new Array(16));
 // }
 
-test('Tile3D#destroys', (t) => {
-  const tile = new Tile3D(MOCK_TILESET, TILE_HEADER_WITH_BOUNDING_SPHERE, undefined);
+test('Tile3D#destroys', async (t) => {
+  const tilesetJson = await load(TILESET_URL, Tiles3DLoader);
+  const tileset = new Tileset3D(tilesetJson);
+  tileset.lodMetricValue = 2;
+
+  const tile = new Tile3D(tileset, TILE_HEADER_WITH_BOUNDING_SPHERE, '');
   t.equals(tile.isDestroyed(), false);
   tile.destroy();
   t.equals(tile.isDestroyed(), true);
   t.end();
 });
 
-test('Tile3D#throws if boundingVolume is undefined', (t) => {
+test('Tile3D#throws if boundingVolume is undefined', async (t) => {
+  const tilesetJson = await load(TILESET_URL, Tiles3DLoader);
+  const tileset = new Tileset3D(tilesetJson);
   const tileWithoutBoundingVolume = clone(TILE_HEADER_WITH_BOUNDING_SPHERE, true);
+
   delete tileWithoutBoundingVolume.boundingVolume;
-  t.throws(() => new Tile3D(MOCK_TILESET, tileWithoutBoundingVolume, undefined));
+  t.throws(() => new Tile3D(tileset, tileWithoutBoundingVolume, ''));
   t.end();
 });
 
-test('Tile3D#throws if boundingVolume does not contain a sphere, region, or box', (t) => {
+test('Tile3D#throws if boundingVolume does not contain a sphere, region, or box', async (t) => {
+  const tilesetJson = await load(TILESET_URL, Tiles3DLoader);
```

Comment:
By depending on the output of load, these are no longer unit tests, but testing depends on a specific loader.

The reason this is an issue is that the tiles module is supposed to be working on a general tile structure that is generated by any tile loader (3d tiles, i3s, potree, entwine, ...), so making the tests dependent on one of those implementations makes less sense.

Did you try to improve the mock tileset instead?

Reply:
I wanted to just avoid type checking errors, but yes we can just disable ts checking.